### PR TITLE
docs(api): Mark aliased exports

### PR DESCRIPTION
**Pull request by martinsik**
#4582 This PR marks renamed exports in the docs app and removes duplicate pages. For example `flatMap` doc is now not a separate page and is also removed from https://rxjs.dev/api/operators. It's just mentioned on https://rxjs.dev/api/operators/mergeMap as an alias for `mergeMap`.

@JWO719 At the end I didn't even have to use `typescript` as I wanted here https://github.com/ReactiveX/rxjs/issues/4582#issuecomment-472821147 because everything is already parsed by `dgeni-packages` typescript processor, it's just not mentioned anywhere https://github.com/angular/dgeni-packages/blob/master/typescript/src/processors/readTypeScriptModules/index.ts#L95-L99.
Also I wrote this processor in TypeScript and then updated `package.json` scripts with `ts-node` because it already works nicely without any other changes with the existing JavaScript codebase. I can eventually rewrite it to JavaScript like the rest of the code.

### CODE REVIEW

**Comment by martinsik** [comment]
@JWO719 That's really weird. I tried removing `docs_app/node_modules` and running `npm i` and then `npm run docs` and it worked. 
In the path you have `C:\dev\rxjs\docs_app\node_modules.bin\dgeni:2` it looks like it's missing one `\` in `node_modules.bin`

**Comment by JWO719** [comment]
I will retry and come back to you

**Comment by benlesh** [comment]
Ping @JWO719

**Comment by martinsik** [comment]
@JWO719 I just saw exactly the same issue in a different project but this happens only on Windows machines. There's something wrong with using paths to `node_modules/.bin`. Can you try replacing the `docs` script with this line?

```
"docs": "npx ts-node -P tsconfig.docs.json ./node_modules/dgeni/lib/gen-docs.js ./tools/transforms/angular.io-package && npm run docs-decision-tree",
```

**Comment by JWO719** [comment]
Thanks for the hint, I will try this afternoon!

**Comment by JWO719** [comment]
the new command solved the issue described above, but unfortunately I now run into:
```
Error: Cannot find module './processors/markAliases'
[0]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
[0]     at Function.Module._load (internal/modules/cjs/loader.js:507:25)
[0]     at Module.require (internal/modules/cjs/loader.js:637:17)
[0]     at require (internal/modules/cjs/helpers.js:22:18)
[0]     at Object.<anonymous> (C:\dev\rxjs\docs_app\tools\transforms\angular-api-package\index.js:36:14)
[0]     at Module._compile (internal/modules/cjs/loader.js:689:30)
[0]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
[0]     at Module.load (internal/modules/cjs/loader.js:599:32)
[0]     at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
[0]     at Function.Module._load (internal/modules/cjs/loader.js:530:3)

```

**Comment by benlesh** [comment]
Any new status here? :)

**Comment by martinsik** [comment]
@JWO719 Sorry for such late response. I think I'll need to reach out to you on Slack or gitter because I don't know why it wouldn't work for you.

File: docs_app/package.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
     "test-pwa-score": "node scripts/test-pwa-score",
     "deploy-production": "scripts/deploy-to-firebase.sh",
     "payload-size": "scripts/payload.sh",
-    "docs": "dgeni ./tools/transforms/angular.io-package && npm run docs-decision-tree",
+    "docs": "npx ts-node -P tsconfig.docs.json ./node_modules/dgeni/lib/gen-docs.js ./tools/transforms/angular.io-package && npm run docs-decision-tree",
     "docs-watch": "node tools/transforms/authors-package/watchr.js",
```
Comment (JWO719) [inline docs_app/package.json]: I now finally noticed what was missing for me. I used the 'serve-and-sync' task which is using the 'docs-watch' command. This isn't compiling the ts code for now. Could you maybe add this here to make it fully working.

Comment (martinsik) [reply docs_app/package.json]: @JWO719 I finally fixed the `docs-watch` script.

**Comment by JWO719** [comment]
I'm really sorry that it took so long, but I'm really happy to finally merged it. Many thanks for your contribution

**Pull request merged**